docs/reference/commandline/ps.md

```diff
@@ -59,7 +99,66 @@ The currently supported filters are:
     106ea823fe4e        fedora:latest     /bin/sh -c 'bash -l'   2 weeks ago         Exited (0) 2 weeks ago                              determined_albattani
     48ee228c9464        fedora:20         bash                   2 weeks ago         Exited (0) 2 weeks ago                              tender_torvalds
 
-This shows all the containers that have exited with status of '0'
+This shows all the containers that have exited successfully, i.e., with exited code of '0'.
+
+#### Status
+
+The `status` filter allows you to list containers filtered on the status of the container. You can filter using `created`, `restarting`, `running`, `paused` and `exited`.
+
+    $ docker ps --filter status=running
+    CONTAINER ID        IMAGE                  COMMAND             CREATED             STATUS              PORTS               NAMES
+    715ebfcee040        busybox                "top"               16 minutes ago      Up 16 minutes                           i_am_nostalgic
+    d5c976d3c462        busybox                "top"               23 minutes ago      Up 23 minutes                           top
+    9b6247364a03        busybox                "top"               24 minutes ago      Up 24 minutes                           nostalgic_stallman
+
+This will list all containers that are running.
+
+    $ docker ps --filter status=paused
+    CONTAINER ID        IMAGE               COMMAND             CREATED             STATUS                      PORTS               NAMES
+    673394ef1d4c        busybox             "top"               About an hour ago   Up About an hour (Paused)                       nostalgic_shockley
+
+This will list all containers that are paused.
+
+#### Ancestor
+
+The `ancestor` filter allows to filter containers based on the image or a descendant of it.
```

Comment:
The `ancestor` filter matches containers based on its image or a descendant of it. For example, to filter for containers that use an `ubuntu` image: